src/components/hero/hero.njk

```diff
@@ -1,7 +1,7 @@
 <section class="usa-hero">
   <div class="usa-grid">
```

Comment:
`usa-grid` will be deprecated, we'll want to use the new grid classes and margins to achieve this effect. What do you think is the best way of doing this @thisisdano?